UnitTests/WatcherManagerTest/StopWatchingTest.cs

```diff
@@ -0,0 +1,68 @@
+﻿using InternshipProject_2.Manager;
+using InternshipProject_2.Models;
+using RequestResponseModels.Watcher.Request;
+using RequestResponseModels.Watcher.Response;
+
+namespace UnitTests
+{
+    [TestClass]
+    public class StopWatchingTest
+    {
+        private WatcherManager _watchManager;
+        private Project2Context _dbContext;
+
+        [TestInitialize]
+        public void Setup()
+        {
+            _dbContext = new Project2Context();
+            _watchManager = new WatcherManager(_dbContext);
+        }
+
+        [TestMethod]
+        public async Task StopWatching_Successful()
+        {
+            //Arrange
+            var user = new User
+            {
+                Username = "Test",
+                Password = "password",
+                Email = "email",
+                Role = "developer",
+                CreatedAt = DateTime.Now
+
+            };
+            var ticket = new Ticket
+            {
+                Title = "Test",
+                Body = "Test",
+                Type = "Test",
+                Priority = "Test",
+                Component = "Test",
+                ReporterId = 1,
+                CreatedAt = DateTime.Now
+            };
+            _dbContext.Users.Add(user);
```

Comment:
Wich dbContext are you using while unit testing?
Not a good idea to add testing data to DB while testing.
You have 2 options:
- use a mock database
- remove the test data after testing